src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -4019,13 +4019,13 @@
     <value>Vertical pane</value>
   </data>
   <data name="AddVerticalPaneDescription" xml:space="preserve">
-    <value>Add vertical pane</value>
+    <value>Add a vertical split pane</value>
   </data>
   <data name="AddHorizontalPane" xml:space="preserve">
     <value>Horizontal pane</value>
   </data>
   <data name="AddHorizontalPaneDescription" xml:space="preserve">
-    <value>Add horizontal pane</value>
+    <value>Add a horizontal split pane</value>
```

Comment:
'Horizonal' refers to the arrangement/orientation of the panes, we should avoid the term 'split' to prevent confusion.